markhor_bringup/launch/markhor_sensors.launch

```diff
@@ -0,0 +1,6 @@
+<launch>
+    <include file ="$(find capra_telaire_ros)/launch/telaire.launch"/>
```

Comment:
On voulait pas changer le nom de ca? Ce n'est plus un telaire qu'on utilise. Ca reste une option qu'on change de CO2 sensor vu qu'il est assez de base celui qu'on a mais pour le moment il marche. Bref, quelquechose de plus "générale" peut être comme nom ?

Reply:
This will be modified in a future PR since the CO2 launch file have the same name right now.